### PR TITLE
Fixes for current version and upgrades

### PR DESCRIPTION
**Pull request by armandomiani**

### CODE REVIEW

**Comment by armandomiani** [comment]
WIP. Please do not review now.

**Comment by armandomiani** [comment]
Changes based on upstream repo: https://github.com/docker-library/cassandra/blob/master/3.11/Dockerfile

File: versions.yaml [hunk 1]
```diff
@@ -20,40 +20,42 @@ versions:
...
+      sha512: db2026342e876caf790833d49f7ab1a2fbba39bf380384ef66e2da4913f537690a56c97cb2f6ea17f667a0d34aeb406fa658db02aec1121a5ba7134ab59a5cfb
+      gpg: 514A2AD631A57A16DD0047EC749D6EEC0353B12C A26E528B271F19B9E5D8E19EA278B781FE4B2BDA A4C465FEA0C552561A392A61E91335D77E3E87CB
+      version: 2.2.16
     gosu:
       gpg: B42F6819007F00F88E364FD4036A9C25BF357DD4
       version: '1.10'
   repo: cassandra2
   tags:
-  - 2.2.15-debian9
+  - 2.2.16-debian9
```
Comment (ovk6) [inline versions.yaml:52]: Revert tags order, it's generated automatically

Comment (armandomiani) [reply versions.yaml:52]: Fixed

**Review by ovk6 — APPROVED** [review]

**Review by vcanaa — APPROVED** [review]

**Pull request merged**